x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_details_page/components/agent_logs/agent_logs.tsx

```diff
@@ -119,6 +120,8 @@ export const AgentLogsUI: React.FunctionComponent<AgentLogsProps> = memo(
   ({ agent, agentPolicy, state }) => {
     const { data, application, http } = useStartServices();
     const { update: updateState } = AgentLogsUrlStateHelper.useTransitions();
+    const config = useConfig();
+    const isLogsUIAvailable = !config.internal?.registry?.capabilities;
```

Comment:
Should we use the cloud plugin `isServerlessEnabled` method instead of that condition it could be more robust, wdyt?

Reply:
updated, I tested by setting `xpack.cloud.serverless.project_id: "1234"` in serverless.yml, as the flag is based on that